tests/test_input_outputs.py

```diff
@@ -24,6 +23,11 @@
 NUM_CLASSES = 2
 
 
+class MultiInputTransformer(FunctionTransformer):
+    def get_meta_params(self):
```

Comment:
Style nit: a better name might be `metadata`.

Reply:
Changed name